ci-operator/step-registry/ipi/conf/nutanix/realtime/ipi-conf-nutanix-realtime-ref.yaml

```
@@ -0,0 +1,13 @@
+ref:
+  as: ipi-conf-nutanix-realtime
+  from_image:
```

```suggestion
  from: cli
```
If not necessary, we can use cli image instead of upi-installer.

---

What's the different of the two? Does it mean use upi-installer only for UPI jobs?